frontend/src/js/components/workspace/WorkspacesSidebarItem.tsx

```diff
@@ -17,19 +20,33 @@ type PropTypes = ReturnType<typeof mapDispatchToProps>
     & ReturnType<typeof mapStateToProps>
     & PropsFromParent
 
-const WorkspacesSidebarItem: FC<PropTypes> = ({selectedEntries, moveItems, selectedWorkspaceId, linkedToWorkspaceId, linkedToWorkspaceName}) => {
-    return <SidebarSearchLink
-        onDrop={(e: React.DragEvent) => {
-            const json = e.dataTransfer.getData('application/json');
-            const {id: idOfDraggedEntry} = JSON.parse(json);
-            const entryIds = getIdsOfEntriesToMove(selectedEntries, idOfDraggedEntry);
-            moveItems(selectedWorkspaceId, entryIds, linkedToWorkspaceId);
-        }}
-        key={linkedToWorkspaceId}
-        to={`/workspaces/${linkedToWorkspaceId}`}
-    >
-        <div className='sidebar__item__text'>{linkedToWorkspaceName}</div>
-    </SidebarSearchLink>
+
+const WorkspacesSidebarItem: FC<PropTypes> = ({selectedEntries, moveItems, copyItems, selectedWorkspaceId, linkedToWorkspaceId, linkedToWorkspaceName}) => {
```

Comment:
There's an edge case that already exists where `selectedWorkspaceId` == `linkedToWorkspaceId` which probably means that the user messed up and didn't actually want to copy or move items to the workspace where they're already located. Could be worth either doing nothing or showing an error modal in this case?

Reply:
Great shout - here's the new modal 
<img width="753" alt="Screenshot 2024-01-09 at 17 54 44" src="https://github.com/guardian/giant/assets/3606555/7f763d71-2481-4adf-8c34-ba725e9b1f2a">